docs/_snippets/setup-teams-integration.mdx

```diff
@@ -0,0 +1,108 @@
+First create a Microsoft Teams team:
+
+<Accordion title="Create a new Team">
```

Comment:
Yeah was not really sure how it worked so did the most important part I think. Also think it would be good to modify the layout a bit that it is less focussed on Slack and more on the Alert part, and from there on that Slack and Teams are available!